sys-power/tlp/tlp-1.9.0-r1.ebuild

```diff
@@ -76,6 +74,30 @@ pkg_postinst() {
 	optfeature "see disk drive health info in tlp-stat" sys-apps/smartmontools
 	optfeature "Sleep hooks" sys-auth/elogind sys-apps/systemd
 	optfeature "Battery functions for ThinkPads prior to the Sandy Bridge generation (2011)" app-laptop/tp_smapi
+
+	if has_version "sys-power/power-profiles-daemon" && use pd; then
+		ewarn
+		ewarn "sys-power/power-profiles-daemon is installed, but is "
+		ewarn "incompatible with tlp-pd daemon. For best results, "
+		ewarn "uninstall one of these packages or set use tlp[-pd]."
```

Comment:
Ha! This proves my point. You meant -ppd there. Rename the flag in tlp please...